sql/core/src/main/scala/org/apache/spark/sql/execution/objects.scala

```diff
@@ -17,6 +17,8 @@
 
 package org.apache.spark.sql.execution
 
+import com.sun.org.apache.xalan.internal.xsltc.compiler.util.VoidType
```

Comment:
Is this a mistaken import? I don't see it used in the change and can't imagine we'd be invoking Xalan here